<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>NRNB: About</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">

    <!-- Le styles -->
    <link href="assets/css/bootstrap.css" rel="stylesheet">
    <link href="assets/css/bootstrap-responsive.css" rel="stylesheet">
    <link href="assets/css/nrnb.css" rel="stylesheet">

    <!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Fav and touch icons -->
    <link rel="shortcut icon" href="assets/ico/favicon.ico">
    <link rel="apple-touch-icon-precomposed" sizes="144x144"
          href="assets/ico/apple-touch-icon-144-precomposed.png">
    <link rel="apple-touch-icon-precomposed" sizes="114x114"
          href="assets/ico/apple-touch-icon-114-precomposed.png">
    <link rel="apple-touch-icon-precomposed" sizes="72x72"
          href="assets/ico/apple-touch-icon-72-precomposed.png">
    <link rel="apple-touch-icon-precomposed"
          href="assets/ico/apple-touch-icon-57-precomposed.png">
</head>

<body>

<div class="belt white">
    <div class="container">
        <div class="section">

            <h3>What is Network Biology?</h3>

            <p>We are all familiar with some of the components of biological systems – DNA, proteins, 
                cells, organs, individuals – but understanding biological systems involves more than 
                just cataloging its component parts. It is critical to understand the many interactions 
                of these parts within systems, and how these systems give rise to biological functions 
                and responses and determine states of health and disease. The National Resource for 
                Network Biology provides the scientific community with a broad platform of computational 
                tools for the study of biological networks and for incorporating network knowledge in 
                biomedical research.</p>
        </div>


        <ul class="nav nav-tabs" id="tabs">
            <li class="active"><a href="#research-tab" data-toggle="tab">Research</a></li>
            <li><a href="#community-tab" data-toggle="tab">Community</a></li>
        </ul>
        <div class="tab-content">
            <div class="tab-pane active" id="research-tab">

                <p>We are planning new technology research projects focused on three complementary themes for which NRNB and its 
                    collaborators are well-positioned to catalyze major change (below in bold). These three themes 
                    serve as the major organizing principles underlying each of the three project areas. All 
                    three projects capitalize on the rapidly increasing accumulation of molecular network data 
                    in mammals, and the concomitant growth of new ‘omics profiling technologies which creates 
                    remarkable new opportunities for using network-based approaches to understand disease states 
                    on a genome-scale in human individuals.
                </p>
                <p><b>1. Generating Differential and Dynamic Networks.</b> New mass spectrometry technology is 
                    making it possible to capture comprehensive changes in protein expression and phosphorylation 
                    at lower cost, higher speed and higher spatial resolution, which makes it possible to measure 
                    differential network expression information in clinical samples and across tissues and time 
                    points. Single-cell genomics, including single-cell RNA-seq, now achieves high resolution 
                    measurements of transcriptional state on a per cell basis over multiple time points, enabling 
                    differential and dynamic network expression information to be measured or inferred. In this 
                    area, we develop new computational technologies that take advantage of these qualitatively 
                    new data types to better understand how networks function in differential biological conditions, 
                    such as cell types and disease states, and to infer whole-cell dynamic network models. The 
                    goals of these technologies are to [1] capture the molecular information flow from targeted 
                    perturbations to downstream cellular responses; [2] functionally characterize 
                    mechanisms defining individual cell types and their positions in developmental lineages; and 
                    [3] capture and visualize differential and dynamic changes in protein interactions 
                    across biological contexts, such as disease versus normal tissue types. </p>
                <p><b>2. Modeling Multi-scale Network Architecture.</b> In this project, we advance methods to 
                    transition Network Biology from flat diagrams of nodes and edges towards multi-scale models 
                    of biological systems. Although current network models and layouts provide a useful summary 
                    of an interaction dataset, these models and visualizations do not capture the exquisite multi-scale 
                    hierarchy of modular components and subcomponents that are evident in many biological systems – 
                    from amino acids to proteins to protein complexes to biological processes to organelles, cells, 
                    and tissues. Previously, we demonstrated that detailed hierarchical information can be systematically 
                    revealed by biological network modelling. This discovery enabled us to reconstruct and greatly 
                    extend the Gene Ontology (GO) hierarchy, yielding a GO built directly from protein networks 
                    (Data-Driven Ontologies) rather than literature. Here we seek to significantly increase the 
                    resolution and scalability of algorithms for detection of hierarchical network structure 
                    [1] and the ability to integrate (embed) many different lines of network evidence 
                    in discovering such structure [2]. We also aim to broaden and generalize the concepts 
                    of hierarchical network analysis to study biological structure at the larger scales of cell 
                    populations and tissues [3]. </p>
                <p><b>3. Network-guided Machine Learning for Biomedicine.</b> Powerful machine learning technologies, 
                    such as the recent advances in deep learning, promise to revolutionize our ability to make 
                    predictions in biology and medicine and could one day replace people in tasks such as image 
                    analysis and prediction of patient drug response. However, machine learning models are typically 
                    black boxes that are challenging to use to gain the mechanistic understanding required for us to 
                    control and fix biological systems for industrial and medical applications. How can we reconcile 
                    these approaches to gain both the predictive power of machine learning and the interpretability 
                    of our mechanistic models of biology? Here we explore a series of complementary and innovative 
                    approaches to this question based on integrating machine learning models with biological networks. 
                    Specifically, we aim to use networks to: [1] Guide the transfer of predictive models of 
                    drug response from model systems to patients; [2] Apply machine learning models to 
                    genotype-phenotype prediction in genome-wide association studies; and [3] Use machine 
                    learning for patient diagnosis and clinical trial selection in precision medicine applications. 
                </p>

            </div>
            <div class="tab-pane" id="community-tab">
                <div class="row">
                    <div class="span9">
                        <p>To serve rapidly growing community of network biology researchers, the NRNB runs a very 
                            active program of Community Engagement. This program includes Training, Dissemination, and 
                            Service components (below in bold). While the community of researchers benefiting from network biology is already 
                            substantial, one suspects that many other investigators (basic, clinical, and 
                            pharmaceutical) as of yet have little knowledge of recent developments in network biology 
                            and would benefit greatly from community engagement centered on NRNB technologies. As our 
                            engagement efforts continue, the network biology community could potentially become a good 
                            deal larger.</p>
                        <p><b>Training.</b> We regularly teach application of NRNB technologies in multiple formats 
                            and in diverse research areas. Each year we lead over a dozen workshops and seminars 
                            across multiple cities and countries. We have matched over 100 students with network biology 
                            mentors as part of Google’s Summer of Code (GSoC) program, for diverse set of successfully 
                            completed projects. We have also offered a full university course in Network Biomedicine 
                            at UCSD which has been very well received by the students. We are planning to expand our education and training platforms to be more extensible and modular, to scale 
                            to a substantially larger community of trainees, and to recruit and empower community 
                            members to become trainers themselves. </p>
                        <p><b>Dissemination.</b> We have established a collection of interlinked websites, social 
                            media feeds and community forums for the dissemination of NRNB technologies and to provide 
                            general information about the field of Network Biology. Regular activities include 
                            prolific posts on social media of late-breaking network biology publications and results 
                            and the establishment of a network biology helpdesk, a professional LinkedIn group, and a 
                            portfolio of tutorial YouTube videos. We launched the NetBio Community of Special Interest 
                            which has recently been promoted to a central track of the Intelligent Systems for Molecular 
                            Biology (ISMB) annual international conference. Our NRNB software ecosystem 
                            provides a major means of dissemination for bioinformatic tools, with connections to platforms 
                            like Cytoscape and Jupyter/Python/R. </p>
                        <p><b>Service.</b> We respond to 60-80 requests for collaboration and service in any given year. 
                            These service collaborations range from assisting users with the application of methods and 
                            tools for network analysis to consulting on the development of Cytoscape apps. Service 
                            activities naturally provide access to, and facilitate adoption of, many center-developed tools.
                        </p>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>

<footer></footer>

<!-- For Navbar -->
<script src="assets/js/jquery-1.11.0.min.js"></script>
<script src="assets/js/bootstrap.min.js"></script>
<script src="assets/js/page_setup.js"></script>

<script type="text/javascript">
    var _gaq = _gaq || [];
    _gaq.push([ '_setAccount', 'UA-19386719-1' ]);
    _gaq.push([ '_trackPageview' ]);

    (function () {
        var ga = document.createElement('script');
        ga.type = 'text/javascript';
        ga.async = true;
        ga.src = ('https:' == document.location.protocol ? 'https://ssl'
                : 'http://www')
                + '.google-analytics.com/ga.js';
        var s = document.getElementsByTagName('script')[0];
        s.parentNode.insertBefore(ga, s);
    })();
</script>

</body>
</html>
